skimage/registration/__init__.py

```diff
@@ -1,8 +1,44 @@
+"""
+Image registration module
+
+This module provides a set of functions to register (align two images into single coordinate system) based on various transformation models. Implemented algorithms include phase_cross_correlation and optical flow estimator.
+
+Functions
+---------
+phase_cross_correlation: function
+Perform the phase correlation to determine the shift between two images using DFT upsampling.
+
+optical_flow_tvl1: function
+Estimates the optical flow components for each axis between two images using TV-L1 solver.
+
+optical_flow_ilk: function
+Estimates the optical flow between two images using iterative Lucas-Kanade(iLK) solver.
```

Comment:
No need to list these functions in the docstring. [In our HTML documentation](https://output.circle-artifacts.com/output/job/bce13a80-1122-4b28-9348-9227cd6d58a4/artifacts/0/doc/build/html/api/skimage.registration.html), the docstring is followed by an automatically generated list. And IDEs usually make these contents discover-able through autocompletion.

Reply:
Alright, do I need to add just the general purpose of these functions in the extended summary?

Reply:
Yeah, it would be good to help users who are discovering this module and are asking themselves "okay, restoration, what is it, why is it useful, and what is it used for".